src/main.ts

```diff
@@ -11,7 +12,11 @@ async function bootstrap() {
 
   applyDocs(app);
 
-  await app.listen(3000);
+  const port = new AppConfig().build().metaData.port;
+
+  console.info(`The app port is ${port}. It's successfully run`);
+
+  await app.listen(port);
 }
```

Comment:
await app.listen(port, () => {
  console.info(`The app is up and running on ${port} port`);
});